src/bootloader.c

```diff
@@ -392,6 +426,8 @@ void LuosBootloader_Task(void)
         case BOOTLOADER_STOP_STATE:
             // save boot_mode in flash
             LuosHAL_SetMode(APPLICATION_MODE);
+            // wait for the command to be send to all nodes
+            LuosHAL_Delay(1000);
```

Comment:
do you still have LuosHAL delay?

Reply:
This has been resolved in commit cca290a